code/cppRawFtoaFunction/ftoa.h

```diff
@@ -0,0 +1,46 @@
+
+#ifndef FTOA_H
+#define FTOA_H
+
+#include <cstdlib>
+#define MAX_STR_LEN 16
+
+/*
+ * This function returns a char array (string) representation of a floating point value
+ * Parameters: d = double to be converted | precision = number of digits after decimal point
+ */
+char *ftoa(double d, int precision) {
```

Comment:
Extremely minor comment. I think the name ftoa is a bad name for a function simple because you do not know what it does simple by reading it. But again minor please ignore if it's just me ;)

Reply:
It's called ftoa (short for float to array) because there is a similarly named function itoa (int to array)